content/cloud-docs/api-docs/providers.mdx

```diff
@@ -336,6 +726,559 @@ curl \
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "provpltfrm-BLJWvWyJ2QMs525k",
+    "type": "registry-provider-platforms",
+    "attributes": {
+      "os": "linux",
+      "arch": "amd64",
+      "filename": "terraform-provider-aws_3.1.1_linux_amd64.zip",
+      "shasum": "8f69533bc8afc227b40d15116358f91505bb638ce5919712fbb38a2dec1bba38",
+      "permissions": {
+        "can-delete": true,
+        "can-upload-asset": true
+      },
+      "provider-binary-uploaded": false
+    },
+    "relationships": {
+      "registry-provider-version": {
+        "data": {
+          "id": "provver-y5KZUsSBRLV9zCtL",
+          "type": "registry-provider-versions"
+        }
+      }
+    },
+    "links": {
+      "provider-binary-upload": "https://archivist.terraform.io/v1/object/dmF1b..."
+    }
+  }
+}
+
+```
+
+## Get all platforms for a single version (private provider)
+
+`GET /organizations/:organization_name/registry-providers/:registry_name/:namespace/:name/versions/:version/platforms`
+
+### Parameters
+
+| Parameter            | Description                                                                                              |
+| -------------------- | -------------------------------------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization the provider belongs to.                                                    |
+| `:namespace`         | The namespace of the provider. Must be the same as the `organization_name` for the provider              |
+| `:name`              | The provider name.                                                                                       |
+| `:version`           | The version of the provider.                                                                             |
+| `:registry_name`     | Must be `private`.                                                                                       |
+
+| Status  | Response                                             | Reason                                                    |
+| ------- | ---------------------------------------------------- | --------------------------------------------------------- |
+| [200][] | [JSON API document][] (`type: "registry-providers"`) | The request was successful                                |
+| [403][] | [JSON API error object][]                            | Forbidden - public provider curation disabled             |
+| [404][] | [JSON API error object][]                            | Provider not found or user unauthorized to perform action |
+
+### Sample Request 
+
+```shell
+curl \
+  --request GET \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms
+```
+
+### Sample Response
+
+```json
+{
+  "data": [
+    {
+      "id": "provpltfrm-GSHhNzptr9s3WoLD",
```

Comment:
Maybe only 2 or 3 platforms in the example output to make it shorter?

Reply:
yeah def a good idea to shorten